<ui version="4.0" >
 <class>GenericOptsForm</class>
 <widget class="QWidget" name="GenericOptsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelURI" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Destination:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLineEdit" name="lineURI" />
   </item>
   <item row="0" column="4" >
    <widget class="QToolButton" name="toolBrowse" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Down speed limit (kiB/s):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QSpinBox" name="spinDown" >
     <property name="maximum" >
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Up speed limit (kiB/s):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QSpinBox" name="spinUp" >
     <property name="maximum" >
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>151</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>52</width>
       <height>121</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
